agent/src/jvmMain/kotlin/com/epam/drill/agent/websocket/WsClientEndpoint.kt

```diff
@@ -27,16 +27,33 @@ import mu.KLogger
 import mu.KotlinLogging
 import com.epam.drill.common.message.Message
 
+
+abstract class RetentionCallbacks() : Endpoint(), MessageHandler.Whole<ByteArray> {

```

Comment:
Convert to interface, move to sepearate file

Reply:
done